springboot/src/main/java/org/infinispan/Application.java

```diff
@@ -0,0 +1,29 @@
+package org.infinispan;
+
+import infinispan.autoconfigure.InfinispanCacheConfigurer;
+import org.infinispan.configuration.cache.CacheMode;
+import org.infinispan.configuration.cache.Configuration;
+import org.infinispan.configuration.cache.ConfigurationBuilder;
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.context.annotation.Bean;
+
+@SpringBootApplication
+public class Application {
```

Comment:
The simple tutorials should be implemented in 1, max 2 files. Could you please put the configuration as well as application code into a single class?